tests/storage/memorydump.go

```diff
@@ -499,93 +409,22 @@ var _ = SIGDescribe("Memory dump", func() {
 
 		It("[test_id:8501]Run memory dump with pvc too small should fail", func() {
 			By("Trying to get memory dump with small pvc")
-			memoryDumpSmallPVC = libstorage.CreateFSPVC(memoryDumpSmallPVCName, testsuite.GetTestNamespace(vm), "200Mi", nil)
-			commandAndArgs := []string{commandMemoryDump, "get", vm.Name, fmt.Sprintf(virtCtlClaimName, memoryDumpSmallPVCName), virtCtlNamespace, vm.Namespace}
-			memorydumpCommand := clientcmd.NewRepeatableVirtctlCommand(commandAndArgs...)
-			Eventually(func() string {
-				err := memorydumpCommand()
-				return err.Error()
-			}, 10*time.Second, 2*time.Second).Should(ContainSubstring("should be bigger then"))
-		})
-	})
-
-	Context("Memory dump with creating a PVC", func() {
-		var (
-			vm *v1.VirtualMachine
-			sc string
-		)
-		const (
-			numPVs = 1
-		)
-
-		BeforeEach(func() {
-			var exists bool
-			sc, exists = libstorage.GetRWOFileSystemStorageClass()
-			if !exists {
-				Skip("Skip no filesystem storage class available")
-			}
-			libstorage.CheckNoProvisionerStorageClassPVs(sc, numPVs)
-
-			vm = createAndStartVM()
-		})
-
-		AfterEach(func() {
-			if vm != nil {
-				deleteVirtualMachine(vm)
-			}
-			pvc, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.NamespaceTestDefault).Get(context.Background(), memoryDumpPVCName, metav1.GetOptions{})
-			if err == nil && pvc != nil {
-				deletePVC(pvc)
-			}
-
-			pvc, err = virtClient.CoreV1().PersistentVolumeClaims(testsuite.NamespaceTestDefault).Get(context.Background(), memoryDumpPVCName2, metav1.GetOptions{})
-			if err == nil && pvc != nil {
-				deletePVC(pvc)
-			}
-		})
-
-		It("[test_id:9034]Should be able to get and remove memory dump", func() {
-			previousOutput := createMemoryDumpAndVerify(vm, memoryDumpPVCName, noPreviousOutput, memoryDumpVirtctlCreatePVC)
-			removeMemoryDumpAndVerify(vm, memoryDumpPVCName, previousOutput, removeMemoryDumpVirtctl)
-		})
-
-		It("[test_id:9035]Run multiple memory dumps", func() {
-			previousOutput := ""
-			for i := 0; i < 3; i++ {
-				By("Running memory dump number: " + strconv.Itoa(i))
-				if i > 0 {
-					// Running memory dump to the same pvc doesnt require claim name
-					memoryDumpVirtctl(vm.Name, vm.Namespace, noClaimName)
-				} else {
-					memoryDumpVirtctlCreatePVC(vm.Name, vm.Namespace, memoryDumpPVCName)
-				}
-				waitAndVerifyMemoryDumpCompletion(vm, memoryDumpPVCName)
-				verifyMemoryDumpNotOnVMI(vm, memoryDumpPVCName)
-				pvc, err := virtClient.CoreV1().PersistentVolumeClaims(testsuite.NamespaceTestDefault).Get(context.Background(), memoryDumpPVCName, metav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				previousOutput = verifyMemoryDumpOutput(pvc, previousOutput, false)
-			}
-
-			removeMemoryDumpAndVerify(vm, memoryDumpPVCName, previousOutput, removeMemoryDumpVirtctl)
-		})
-
-		It("[test_id:9036]Run memory dump to creates a pvc, remove and run memory dump to create a different pvc", func() {
```

Comment:
I think 1 of this 3 tests that uses the virtctl with the create flag should stay (either 9034 9035 9036 maybe the one with multiple memory dumps to verify no issue with the creation of the pvc)

Reply:
These unit tests verify that a PVC was created if `virtctl memorydump` was asked to.

To me that suffices to verify virtctl is doing the right thing (creating a PVC and calling the memorydump subresource with the PVC's name).

https://github.com/kubevirt/kubevirt/blob/715b28f03422da3d930afce38bc90a3e657e7cf0/pkg/virtctl/memorydump/memorydump_test.go#L212
https://github.com/kubevirt/kubevirt/blob/715b28f03422da3d930afce38bc90a3e657e7cf0/pkg/virtctl/memorydump/memorydump_test.go#L226
https://github.com/kubevirt/kubevirt/blob/715b28f03422da3d930afce38bc90a3e657e7cf0/pkg/virtctl/memorydump/memorydump_test.go#L242